internal/db/mailbox.go

```diff
@@ -303,42 +317,33 @@ func FilterMailboxContains(ctx context.Context, client *ent.Client, mboxID imap.
 	return result, nil
 }
 
-func getGlobalUIDValidity(ctx context.Context, tx *ent.Tx) (imap.UID, error) {
-	globalUIDValidity, err := tx.UIDValidity.Query().
-		Select(uidvalidity.FieldUIDValidity).
-		Only(ctx)
-
-	if ent.IsNotFound(err) {
-		_, err := tx.UIDValidity.Create().
-			SetUIDValidity(mailbox.DefaultUIDValidity).
-			Save(ctx)
-		if err != nil {
-			return 0, err
+func InitGlobalUIDValidity(ctx context.Context, tx *ent.Tx, uidValidity imap.UID) error {
+	if _, err := tx.UIDValidity.Query().Select(uidvalidity.FieldUIDValidity).Only(ctx); err != nil {
```

Comment:
Should we maybe update the UID validity here if the value reported by the connector is higher than the one we have now?

Reply:
I don't think so; the validity reported by the connector should never be higher than what we have, since we're the only ones ones who increase it. If it is higher, something's gone wrong 🤔

Reply:
What if the user is using an outdated version of the cache that they have left lying around and decide to plug it in now?

Reply:
Yes, but in our current state you can still do this and then have the remote be set to a lower UID validity than what the connector may already have.

Reply:
Good point, we should at least be safe regarding that

Reply:
Done